Proxy task 2

a-prakaptsova opened the pull request:
NASA API request (asteroids per period) and output inаormation to the console have been implemented

anastasiya-pr committed:
Proxy task 2:  NASA API request (asteroids per period) and output information to the console have been implemented

i-siz commented on src/index.js:
```diff
@@ -0,0 +1,40 @@
+const DEFAULT_ENCODING = 'utf8';
+
+const START_DATE = '2024-03-04';
+const END_DATE = '2024-03-08';
```
I think you can calculate the last Monday and Friday dinamically based on the current date with a help of [moment](https://www.npmjs.com/package/moment) library.
Useful examples you can find [here](https://www.geeksforgeeks.org/moment-js-moment-day-method/).

i-siz commented on src/index.js:
```diff
@@ -0,0 +1,40 @@
+const GET_ASTEROIDS_URL = `https://api.nasa.gov/neo/rest/v1/feed?start_date=${START_DATE}&end_date=${END_DATE}&api_key=${API_KEY}`;
+
+const getAllAsteroidsInPeriod = () => {
+    https.get(GET_ASTEROIDS_URL, (res) => {
```
Instead of built-in tools, I can suggest using the [Axios](https://www.npmjs.com/package/axios) library for queries. It makes the code look more concise.

anastasiya-pr committed:
Proxy task 2:
- https library has been changed to axios library
- using moment library for dates
- .gitignore has been added
